src/test/java/divaeva/hw/api/pet/PetTestBase.java

```diff
@@ -0,0 +1,34 @@
+package divaeva.hw.api.pet;
+
+import dto.CategoryDTO;
+import dto.PetDTO;
+import dto.TagDTO;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PetTestBase {
+    protected final String BASE_SERVER_URL_PET = "https://petstore.swagger.io/v2/pet/";
```

Comment:
As I mentioned previously, create enum for all URLS

Reply:
fixed